### PR TITLE
[WIP] Feature Progress bar seek

### PR DESCRIPTION
**Pull request by HSavynetska**
ATF Test Scripts to check #[1735](https://github.com/smartdevicelink/sdl_core/issues/1735)

This PR is **not ready** for review.

### Summary
Test script for checking new feature Feature Progress bar seek.

### ATF version
[Develop branch](https://github.com/smartdevicelink/sdl_atf/tree/develop)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/API/Progress_Bar_Seek_Feature/commonProgressBarSeek.lua [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+--! @parameters:
+--! pValue - value for enableSeek parameter
+--! @return: none
+--]]
+function c.SetMediaClockTimer(pValue)
+  c.requestParams.enableSeek = pValue
+  local cid = c.getMobileSession():SendRPC("SetMediaClockTimer", c.requestParams)
+
+  c.requestParams.appID = c.getHMIAppId()
+  EXPECT_HMICALL("UI.SetMediaClockTimer", c.requestParams)
```
Comment (KrupenichOleg) [inline test_scripts/API/Progress_Bar_Seek_Feature/commonProgressBarSeek.lua]: @HSavynetska, replace _EXPECT_HMICALL(...)_ with _c.getHMIConnection():ExpectRequest(...)_

Comment (HSavynetska) [reply test_scripts/API/Progress_Bar_Seek_Feature/commonProgressBarSeek.lua]: @KrupenichOleg  fixed

File: test_scripts/API/Progress_Bar_Seek_Feature/001_EnableSeek_Success_flow_with_true_and_false.lua [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
```
Comment (dboltovskyi) [inline test_scripts/API/Progress_Bar_Seek_Feature/001_EnableSeek_Success_flow_with_true_and_false.lua]: @HSavynetska Link to incorrect proposal

File: test_scripts/API/Progress_Bar_Seek_Feature/009_OnSetMediaClockTimer_ignored_flow_in_case_SetMediaClockTimer_request_with_enableSeek_false.lua [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+-- 2) Does not send OnSeekMediaClockTimer notification to the mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/Progress_Bar_Seek_Feature/commonProgressBarSeek')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+
```
Comment (dboltovskyi) [inline test_scripts/API/Progress_Bar_Seek_Feature/009_OnSetMediaClockTimer_ignored_flow_in_case_SetMediaClockTimer_request_with_enableSeek_false.lua]: @HSavynetska Extra new line

File: test_scripts/API/Progress_Bar_Seek_Feature/commonProgressBarSeek.lua [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  c.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+--[[ @setMediaClockTimerUnsuccess: Processing SetMediaClockTimer with ERROR resultCode
+--! @parameters:
+--! pValue - value for enableSeek parameter
+--! pResultCode - result error
+--! @return: none
+--]]
+function c.setMediaClockTimerUnsuccess(pValue, pResultCode)
```
Comment (dboltovskyi) [inline test_scripts/API/Progress_Bar_Seek_Feature/commonProgressBarSeek.lua]: @HSavynetska Please use the same name pattern for functions - 1st letter in upper case or lowercase. Now there is a mix: (`SetMediaClockTimer` vs `setMediaClockTimerUnsuccess`)

File: test_sets/smoke_tests.txt [hunk 5]
```diff
@@ -43,6 +43,7 @@
 ./test_scripts/Smoke/API/043_UnsubscribeButton_Non_Media_PositiveCase_SUCCESS.lua
 ./test_scripts/Smoke/API/044_AlertManeuver_Non_Media_PositiveCase_SUCCESS.lua
 ./test_scripts/Smoke/API/045_PerformAudioPassThru_Non_Media_PositiveCase_SUCCESS.lua
+./test_scripts/Smoke/API/046.lua
```
Comment (dboltovskyi) [inline test_sets/smoke_tests.txt]: @HSavynetska Please put correct name for the script

File: test_scripts/Smoke/API/046_OnSeekMediaClockTimer_PositiveCase_SUCCESS.lua [hunk 6]
```diff
@@ -0,0 +1,66 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: OnSeekMediaClockTimer
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [OnSeekMediaClockTimer]: getting SUCCESS:UI.SetMediaClockTimer()
+
```
Comment (dboltovskyi) [inline test_scripts/Smoke/API/046_OnSeekMediaClockTimer_PositiveCase_SUCCESS.lua]: @HSavynetska Please add sections: 
1. splitter (`-------`)
2. Description
3. Pre-conditions
4. Steps
5. Expected

See other scripts in `Smoke/API`

**Pull request merged**